Runtime/Extensions/GameObjectExtensions.cs

```diff
@@ -236,5 +238,18 @@ public static bool IsParentOrChildOf(this GameObject g1, GameObject g2)
             T component = gameObject.GetComponent<T>();
             return component.IsNull() ? gameObject.AddComponent<T>() : component;
         }
+
+        /// <summary>
+        /// Validates the <see cref="GameObject"/> reference.
+        /// </summary>
+        /// <param name="gameobject">The target <see cref="GameObject"/>.</param>
+        /// <param name="callerName">The <see cref="CallerFilePathAttribute"/> fills in this information.</param>
+        public static void Validate(this GameObject gameobject, [CallerFilePath] string callerName = "", string fieldName = "")
```

Comment:
What in the world does this do?

Reply:
Allows you to validate if a GameObject reference has been applied in a script and then warns you if something hasn't been configured and on which object.  so in OnValidate you can do something like this:

```csharp
 myGameObjectReference.Validate();
```

This effectively ensures a gameobject has been assigned for a script.